x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/engine/engine_connect/search_application_api.tsx

```diff
@@ -34,11 +33,21 @@ import { EngineApiIntegrationStage } from './engine_api_integration';
 import { EngineApiLogic } from './engine_api_logic';
 import { GenerateEngineApiKeyModal } from './generate_engine_api_key_modal/generate_engine_api_key_modal';
 
+import { useCloudDetails } from '../../../../shared/cloud_details/cloud_details';
+import { decodeCloudId } from '../../../../shared/decode_cloud_id/decode_cloud_id';
+
+export const elasticsearchUrl = (): string => {
+  const defaultUrl = 'http://localhost:9200';
+  const cloudContext = useCloudDetails();
+  const url =
+    (cloudContext.cloudId && decodeCloudId(cloudContext.cloudId)?.elasticsearchUrl) || defaultUrl;
```

Comment:
@TattdCodeMonkey @joemcelroy Any idea what `defaultUrl` should actually be for an on prem solution?

Reply:
I suggest default being "https://localhost:9200" -  then its consistent with how content indices does it when no cloud host is found.

https://github.com/elastic/kibana/blob/60c82765779419d356a131e212682b69b035804b/x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/components/curl_request/curl_request.tsx#L32

Reply:
👍 Thanks! Will this be sufficient for on prem installs?